ciao-scheduler/scheduler.go

```diff
@@ -453,22 +483,42 @@ func (sched *ssntpSchedulerServer) getWorkloadResources(work *payloads.Start) (w
 	if workload.diskReqMB < 0 {
 		return workload, fmt.Errorf("invalid start payload local disk demand: disk MB (%d) < 0, must be >= 0", workload.diskReqMB)
 	}
-	if workload.networkNode != 0 && workload.networkNode != 1 {
-		return workload, fmt.Errorf("invalid start payload resource demand: network_node (%d) is not 0 or 1", workload.networkNode)
-	}
 
 	// note the uuid
 	workload.instanceUUID = work.Start.InstanceUUID
 
 	return workload, nil
 }
 
+func networkDemandsSatisfied(node *nodeStat, workload *workResources) bool {
+	if node.isNetNode {
```

Comment:
Stylistically it might be nice to change this test to 

```
if !node.isNetNode {
    return true
}
```

Doing so would allow you to get rid of one level of indentation.